src/snowflake/snowpark/modin/plugin/_internal/frame.py

```diff
@@ -93,6 +96,11 @@ class InternalFrame:
     # The length of data_column_index_names equals to number of multiindex levels.
     # For a 3-level MultiIndex, the value can be like ['A', 'B', 'C']
     data_column_index_names: tuple[LabelTuple, ...]
+    # Map from snowflake identifier to cached Snowpark pandas data type.
+    # The type is None if we don't know the Snowpark data type.
+    # n.b. that we map to SnowparkPandasType rather than to DataType, because
+    # we don't want to try tracking regular Snowpark Python types at all.
+    snowflake_quoted_identifier_to_data_type: dict[str, Optional[SnowparkPandasType]]
```

Comment:
```suggestion
    snowflake_quoted_identifier_to_type: dict[str, Optional[SnowparkPandasType]]
```

Reply:
we can be consistent that:
`type` is short for snowpark pandas type;
`pandas type` is the native pandas type;
`snowpark type` is the snowpark python type.